src/content/1.7/development/multistore/configuration-forms/_index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Configuration forms
+menuTitle: Configuration forms
+summary: "Configuration forms when multistore is enabled"
+weight: 50
+---
+
+# Configuration forms in multistore context
+{{< minver v="1.7.8" title="true" >}}
+
+Since Prestashop 1.7.8, when multistore is enabled, migrated configuration forms can display a checkbox before fields in order to specify if the configuration value is set for the current context shop/group, or if it is a value inherited from all contexts, or from a shop group.
+
+If you're developing a configuration form and you want it to take into account the current multistore context, then you must make your form compatible with multistore checkboxes. This is made possible thanks to [Symfony's form extensions mechanism](https://symfony.com/doc/3.4/form/create_form_type_extension.html).
+
+
+
+## Creating a multistore compatible configuration form
+
+A Symfony form extension called MultistoreConfigurationTypeExtension has been created in order to make it easy to add multistore checkboxes to a configuration form.
+
+In order to extend this form extension, your Symfony form type must implement a `getParent()` method returning `MultistoreConfigurationType::class`.
+
+```php
+<?php
+
+    /**
+     * {@inheritdoc}
+     *
+     * @see MultistoreConfigurationTypeExtension
+     */
+    public function getParent(): string
+    {
+        return MultistoreConfigurationType::class;
+    }
+```
+
+Then for each form elements that should have its multistore checkbox, you must add a `multistore_configuration_key` attribute, with the configuration key to which it should be attached as a value:
+
+```php
+<?php
+class MyFormType extends TranslatorAwareType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add(
+                'my_field',
+                SwitchType::class,
+                [
+                    'required' => true,
+                    // Here we add the multistore_configuration_key attribute with the configuration key it is linked to.
+                    'attr' => [
+                        'multistore_configuration_key' => 'THE_CONFIGURATION_KEY',
+                    ],
+                    // ...
+                ]
+            )
+        // ...
+
+```
+
+Last but not least, you must make the frontend aware of these state modifications, you just need to add the `MultistoreConfigField` in your javascript:
```

Comment:
```suggestion
Last but not least, you must make the frontend aware of these state modifications, you need to add the `MultistoreConfigField` in your javascript:
```
May I suggest to avoid words like "just", "simply", "easy" ? 😉 

This is because of https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher